games-roguelike/tome/tome-2.3.5.ebuild

```diff
@@ -1,8 +1,8 @@
 # Copyright 1999-2016 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-inherit eutils games
+EAPI=7
```

Comment:
New line after EAPI. Please also fix the copyright headers (use repoman commit or pkgdev commit).